### ISSUE TITLE
Virtual Scroller

### ISSUE THREAD

**Issue by dregge8** [xaksis/vue-good-table#347]
### Issue Type

- [x] Enhancement Request

### 

Is there any plan to implement a Virtual Scroller?  Would be pretty neat to be able to pull a few thousands rows and just render accordingly as opposed to pagination.  It is becoming quite a popular method of rendering tables and lists.  Please see the following examples.

https://github.com/Akryum/vue-virtual-scroller
https://github.com/tangbc/vue-virtual-scroll-list

**Comment by niallobrien** [comment]
I too would love to see this feature implemented. Thanks in advance.

**Comment by dregge8** [comment]
Yeah its a great feature and I am finding it extremely difficult to implement into my application.  Not many vue scrollers work well with tables, they all seem to have caveats or are overly complex.  If someone with the knowledge can handle this, it will continue to set vue-good-table above the rest!

**Comment by abhinavsingi** [comment]
Would be great to have it - no table currently supports it

**Comment by fobass** [comment]
hi any update?

**Comment by Zac-Zajdel** [comment]
The company I work for extensively uses this package to show data to our users. Has any progress been made on this? This would significantly advance this packages capabilities. We may have to look for other alternatives. Seems as if this has been ignored.

**Comment by SMontiel** [comment]
Hi, any update on this?
